apps/lidar_odometry_step_1/lidar_odometry.cpp

```diff
@@ -324,16 +324,16 @@ void lidar_odometry_gui()
                                        auto calibration = MLvxCalib::CombineIntoCalibration(idToSn, preloadedCalibration);
                                        auto data = load_point_cloud(fn.c_str(), true, params.filter_threshold_xy, calibration);
 
-                                       if (fn == laz_files.front())
-                                       {
-                                           fs::path calibrationValidtationFile = wdp / "calibrationValidation.asc";
+                                    //    if (fn == laz_files.front())
+                                    //    {
+                                    //        fs::path calibrationValidtationFile = wdp / "calibrationValidation.asc";
 
-                                           std::ofstream testPointcloud{calibrationValidtationFile.c_str()};
-                                           for (const auto &p : data)
```

Comment:
Here You can save first 1000.000 ponts

Reply:
We can save it only when multi-lidar setup is loaded.